### PR TITLE
Add profile screen

### PR DESCRIPTION
**Pull request by psakurai**

### CODE REVIEW

File: screens/HomeScreen.js [hunk 1]
```diff
@@ -1,123 +1,79 @@
...
+        placeholder="President of the People's Republic of China & General Secretary of the Communist Party of China."
+        placeholderTextColor="black"
+        autoCapitalize="none"
+        editable={false}
+      />
+      <TouchableOpacity
+        style={styles.button}>
+        <Text style={styles.ButtonText}>Edit Profile</Text>
+      </TouchableOpacity>
+    </View>
```
Comment (mangkoran) [inline screens/HomeScreen.js:26]: From line 5 until line 26 please mind the indentation.

**Review by mangkoran — CHANGES_REQUESTED** [review]

File: screens/HomeScreen.js (hunk 1 above)
Comment (psakurai) [reply screens/HomeScreen.js]: I had changed the code indentation.

Comment (mangkoran) [reply screens/HomeScreen.js]: The `return` line still left with wrong indentation. Please review the whole `return` block again.

Comment (psakurai) [reply screens/HomeScreen.js]: Done. Do I need to add space after line 28?

Comment (mangkoran) [reply screens/HomeScreen.js]: It's better if you could.

**Comment by mangkoran** [comment]
May you snap a screenshot of the screen?

File: screens/HomeScreen.js (hunk 1 above)
Comment (psakurai) [reply screens/HomeScreen.js]: Alright, I think can merge now.

**Comment by psakurai** [comment]
> May you snap a screenshot of the screen?

photo_2022-01-06_21-16-12

**Comment by mangkoran** [comment]
After thinking for a while, could you update the layout based on penpot?

**Comment by mangkoran** [comment]
I noticed that you create this PR based on `develop` branch. From now on, please make PR based on `develop-v2` branch and target `develop-v2` branch.

**Pull request closed without merging**